cbindings/fixed_pedersen.t.cc

```diff
@@ -91,6 +91,40 @@ TEST_CASE("we can compute multi-exponentiations with a fixed set of generators")
     REQUIRE(res[1] == generators[0]);
   }
 
+  SECTION("we can compute a multiexponentiation of varying length") {
+    cbn::reset_backend_for_testing();
+    const sxt_config config = {SXT_GPU_BACKEND, 0};
+    REQUIRE(sxt_init(&config) == 0);
+
+    wrapped_handle h{generators.data(), 2};
+    REQUIRE(h.h != nullptr);
+
+    uint8_t scalars[] = {0b1010, 0b0101};
+    unsigned bit_table[] = {3, 1};
+    unsigned lengths[] = {2, 2};
```

Comment:
Should there be a test for variable `lengths`, like `unsigned lengths[] = {1,2}`?

Reply:
I modified the tests to be of varying length.